package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jellysync",
-  "version": "1.0.1",
+  "version": "1.0.6",
```

Comment:
why the change?

Reply:
U can never publish the same version to npm